scripts/scil_register_tractogram.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+
+from dipy.align.streamlinear import whole_brain_slr
+import nibabel as nib
+from nibabel.streamlines.array_sequence import ArraySequence
+import numpy as np
+
+from scilpy.io.streamlines import ichunk
+from scilpy.io.utils import (add_overwrite_arg, add_verbose_arg,
+                             assert_inputs_exist, assert_outputs_exist)
+
+DESCRIPTION = """
+Generate a linear transformation matrix from the registration of
+2 tractograms. Typically, this script is run before
+scil_apply_transform_to_tractogram.py.
+
+For more informations on how to use the various registration scripts
+see the doc/tractogram_registration.md readme file
+"""
+
+EPILOG = """
+References:
+[1] E. Garyfallidis, O. Ocegueda, D. Wassermann, M. Descoteaux
+Robust and efficient linear registration of white-matter fascicles in the
+space of streamlines, NeuroImage, Volume 117, 15 August 2015, Pages 124-140
+(http://www.sciencedirect.com/science/article/pii/S1053811915003961)
+"""
+
+
+def register_tractogram(moving_filename, static_filename,
+                        only_rigid, amount_to_load, matrix_filename,
+                        verbose):
+
+    amount_to_load = max(250000, amount_to_load)
+    moving_tractogram = nib.streamlines.load(moving_filename, lazy_load=True)
```

Comment:
@frheault Can we use the stateful_tractogram here ?

Reply:
Done!
@frheault can you validate ?